rcgen/src/lib.rs

```diff
@@ -776,6 +794,83 @@ mod tests {
 		}
 	}
 
+	#[test]
+	fn distinguished_name_remove_no_match() {
+		let mut dn = DistinguishedName::new();
+		// Domain Component (DC)
+		let dc_type = DnType::CustomDnType(vec![0, 9, 2342, 19200300, 100, 1, 25]);
```

Comment:
It seems unimportant for the test added in this file that the `DnType` be a `Custom` one for DC. I think you could avoid some duplication by using a predefined `DynType`. WDYT?